starknet_devnet/blueprints/base.py

```diff
@@ -16,9 +16,15 @@ def extract_positive(request_json, prop_name: str):
     if value is None:
         raise StarknetDevnetException(message=f"{prop_name} value must be provided.", status_code=400)
 
-    if not isinstance(value, int):
+    if not isinstance(value, (int, float)):
         raise StarknetDevnetException(message=f"{prop_name} value must be an integer.", status_code=400)
 
+    if isinstance(value, float):
+        if value%1 == 0:
+            value = int(value)
+        else:
+            raise StarknetDevnetException(message=f"{prop_name} value can't be decimal", status_code=400)
```

Comment:
Would it make sense to use the same error message of "value must be an integer". Decimal might be understood as having the base 10.